src/Core/NotificationHub/PushRegistrationData.cs

```diff
@@ -0,0 +1,19 @@
+﻿namespace Bit.Core.NotificationHub;
+
+public class PushRegistrationData
+{
+    public string Token { get; set; }
+    public (string Endpoint, string P256dh, string Auth)? WebPush { get; set; }
```

Comment:
Lets make a named type for `(string Endpoint, string P256dh, string Auth)` I think I've seen it repeated 7 times or so.